rtloader/test/telemetry/telemetry_test.go

```diff
@@ -0,0 +1,197 @@
+package testtelemetry
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/StackVista/stackstate-agent/rtloader/test/helpers"
+)
+
+func TestMain(m *testing.M) {
+	err := setUp()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error setting up tests: %v", err)
+		os.Exit(-1)
+	}
+
+	os.Exit(m.Run())
+}
+
+func TestSubmitTopologyEvent(t *testing.T) {
```

Comment:
Can we add another one that looks like this, but misses some fields. We'll have to see what we consider required in the agent integration schemas